docs/src/components/Toc.js

```diff
@@ -130,42 +134,44 @@ export default function Toc({ cards }: {| cards: Array<Node> |}): Node {
     }
   };
 
-  React.useEffect(
+  useEffect(
     () => () => {
       clearTimeout(unsetClickedRef.current);
     },
     [],
   );
+
   return (
-    <Box>
-      {anchors.map((anchor, index) => (
-        <Box key={index} display="flex">
-          <Box color={activeState === anchor.id ? 'pine' : 'lightGray'} width={1} flex="none" />
-          <Link
-            hoverStyle={activeState === anchor.id ? 'none' : 'underline'}
-            href={`#${anchor.id}`}
-            onClick={({ event }) => handleClick(anchor.id, event)}
-          >
-            <Box padding={2}>
-              {anchor.closest('h2') ? (
-                <Text color={activeState === anchor.id ? 'pine' : 'darkGray'} weight="bold">
-                  {anchor.closest('h2')?.textContent}
-                </Text>
-              ) : (
-                <Box paddingX={3}>
-                  <Text
-                    size="md"
-                    color={activeState === anchor.id ? 'pine' : 'darkGray'}
-                    weight="bold"
-                  >
-                    {anchor.innerText}
+    <Box maxHeight="calc(100% - 60px)" minWidth={240} overflow="auto" position="fixed">
```

Comment:
This is the real change. Taken directly from [Christian's PR](https://github.com/pinterest/gestalt/pull/1166/files)

Reply:
I think it needs some bottom padding 
<img width="526" alt="Screen Shot 2021-02-11 at 12 15 20 PM" src="https://user-images.githubusercontent.com/5125094/107693382-ed5fb780-6c62-11eb-9533-922fe21bf90e.png">

Reply:
https://www.loom.com/share/3e050384c5884a3a94dc52b00b15d1f6

Reply:
good catch!